packages/get-safe-property-chain/.eslintignore

```diff
@@ -0,0 +1,2 @@
+**/*.js
+types
```

Comment:
Why do you need to ignore the `types`? I tested removing this property for every package, run a fresh build, run the `lint` command, and everything is fine. As `type` files aren't included in the `tsconfig.eslint.json`, eslint shouldn't try to lint them

Reply:
Yep. However, if you open a file that is inside the `types` folder, you get the following error:

![image](https://user-images.githubusercontent.com/6247440/123918158-e3292600-d983-11eb-9765-f72311852d1b.png)

It is just to avoid it. If you think this is not necessary, I can remove `types` from the `.eslintignore` file.